modules/claims_api/lib/bd/bd.rb

```diff
@@ -156,6 +156,10 @@ def generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:,
       end
     end
 
+    def find_pctpnt_vet_id(auth_headers, pctpnt_vet_id)
+      pctpnt_vet_id.presence || auth_headers['va_eauth_pid']
+    end
+
```

Comment:
I went with this instead of adding params being sent here from every possibly related job since that has been a little error prone and I felt a catch would be a more robust option now and into the future.